datanode/sqlstore/node.go

```diff
@@ -43,6 +44,7 @@ func NewNode(connectionSource *ConnectionSource) *Node {
 
 func (store *Node) UpsertNode(ctx context.Context, node *entities.Node) error {
 	defer metrics.StartSQLQuery("Node", "UpsertNode")()
+	store.log.Info("Upserting node", zap.String("node", node.ID.String()))
```

Comment:
super minor but I think the convention here is to use logging.String()

Reply:
Sorry, that should have been removed. I will update it.